### PR TITLE
update go version to v1.19.2

### LINKED ISSUE #33: Update go version to v1.19.2

### PR DESCRIPTION
**Pull request by amarps**
related to #33 

Hi, if you don't mind I would be happy if this repo have hacktoberfest topic so my PR will be counted on hacktoberfest.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@ ORG := jenkins-x
 ORG_REPO := $(ORG)/$(NAME)
 RELEASE_ORG_REPO := $(ORG_REPO)
 ROOT_PACKAGE := github.com/$(ORG_REPO)
-GO_VERSION := 1.18.4
+GO_VERSION := 1.19.2
```
Comment (ankitm123) [inline Makefile:16]: Please change this to: https://github.com/jenkins-x/jx/blob/d06edf36217af6eb6ffd7a209245933dabd4cdb3/Makefile#L18

**Comment by ankitm123** [comment]
@amarps I have added the hacktoberfest label to this repo. I have one minor comment, once you change that, it should be good to merge. Thanks for the contribution :)

File: go.mod [hunk 2]
```diff
@@ -46,4 +46,4 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
-go 1.18
+go 1.19
```
Comment (ankitm123) [inline go.mod:49]: no changes from running go mod tidy?

Comment (amarps) [reply go.mod:49]: image
yes there is no changes after running go mod tidy.

**Review by ankitm123 — APPROVED** [review]

**Pull request merged**